### ISSUE TITLE
Page to explain SAM-scale

### ISSUE THREAD

**Issue by EllaSoderberg** [EllaSoderberg/EmotionandLiking#7]
I removed the labels from the SAM-scale so now we need to explain them somewhere else.

**Comment by EllaSoderberg** [comment]
How are we going to explain valence and arousal?

**Comment by Geoodees** [comment]
As an anecdote, I tried to use 'Lust' (swedish word for arousal) to explain the feeling of excitment to a few actors I was working with. However, we had to spend some time to disconnect it from their assumptions.

This is kind of the same issue. I will look through the paper about the SAM-scale again and see if I can come up with something for both meanings.

**Comment by EllaSoderberg** [comment]
Yeah I think to Swedish participants the word "Arousal" will be interpreted as something sexual. I guess "Excitement" in English is a more neutral word for Swedish participants.

**Comment by kvarnberg** [comment]
I also think arousal could be a little hard to define in a clear way, but I saw someone explaining it as it spans from a sleepy to awake state. Or it could be more of how much reaction the stimulus evokes in the person. Either way, excitement seems more suiting than arousal and should probably be mentioned haha.

**Comment by kvarnberg** [comment]
Valence I think is more of positive vs. negative. For example with audio stimulus, does the melody make you feel positive? Or do you interpret it as telling a negative story?

**Comment by Geoodees** [comment]
Ana has descriptions we can use. Let's use them.
